README.md

```diff
@@ -1,135 +1,410 @@
+   Deploying with signer: 0x8849BAbdDcfC1327Ad199877861B577cEBd8A7b6 on network kadena_devnet0
+   Deploying with signer: 0x8849BAbdDcfC1327Ad199877861B577cEBd8A7b6 on network kadena_devnet1
+           ✔ Should return the correct chainweb chain id
+       getCrossChainAddress
+         Success Test Cases
+   Found 2 Kadena devnet networks: kadena_devnet0, kadena_devnet1
+   Deploying with signer: 0x8849BAbdDcfC1327Ad199877861B577cEBd8A7b6 on network kadena_devnet0
+   Deploying with signer: 0x8849BAbdDcfC1327Ad199877861B577cEBd8A7b6 on network kadena_devnet1
+           ✔ Should return the correct cross chain address (5634ms)
+   ...
+   ```
+
+### Restarting the development network
+
+If the development network stops producing blocks or seems stuck, you can restart the chainweb-node service without stopping or restarting other network components.
+
+To restart the development network:
 
-    The network runs a single Chainweb node along with a single mining client.
-    PoW consensus is disabled and blocks are produced at a constant rate of two
-    seconds per per chain, or ten blocks per second across the whole network.
--   A hardhat Solidity project that demonstrates the implementation of a simple
-    ERC-20 token with support for burn-mint style transfers between the EVM
-    chains in the network.
--   An ethers projects that provides BIP-44 wallets for all allocations in the
-    Genesis block.
--   Docker compose project for running [Blockscout](https://blockscout.com)
-    instances for the EVM chains in the development network. Blockscout is a
-    blockchain explorer that provides an user experience and API similar to
-    [Etherscan](https://etherscan.io).
--   A commandline tool for managing and monitoring the different network
-    components in the Kadena EVM sandbox.
+```sh
+./network devnet restart
+```
 
-## System requirements
+### Stopping the development network
 
-For running the development network:
+When you're finished testing, you can shut down the development network, remove all containers, and reset the database to a clean state.
 
--   [docker](https://www.docker.com) or a compatible alternative.
--   [docker compose](https://docs.docker.com/compose/) or a compatible
-    alternative.
--   At least 4 CPU cores and 8GB of Memory that are available to docker. (Please
-    check the docker configuration if it provides sufficient resources to
-    docker.)
--   Network connection for downloading the container images for the network
-    components.
--   Command line terminal with a POSIX compliant shell.
--   `bash` installation.
--   [jq](https://jqlang.org).
+To shut down the network and remove containers:
 
-For deploying the Solidity contracts with hardhat:
+```sh
+./network devnet stop
+```
 
--   A local [Node.js](https://nodejs.org), [npm](https://www.npmjs.com) (or
-    yarn), and [npx](https://docs.npmjs.com/cli/v8/commands/npx) installation.
+## Integrating with other Hardhat projects
```

Comment:
There are a couple of things that developers will need to be aware of when running tests against our devnet in Hardhat that they normall wouldn't have to think about. Is it best if I write something up and send it to you to edit and put where you want?